### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by DmitryMandrik**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>by.minsk.selenium.mandrik</groupId>
+    <artifactId>selenium-project</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-server</artifactId>
+            <version>3.0.1</version>
```
Comment (EugenBorisik) [inline pom.xml]: I always recommend to use the latest versions (3.9.1)

File: src/test/java/tests/Login.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        driver = Chrome.initChromeDriver();
+    }
+
+    @Test
+    public void logIn(){
+        LoginPage loginPage = new LoginPage();
+        driver.findElement(loginPage.email).sendKeys("EugenBorisik");
+        driver.findElement(loginPage.password).sendKeys("qwerty12345");
+        driver.findElement(loginPage.submit).click();
+    }
```
Comment (EugenBorisik) [inline src/test/java/tests/Login.java]: Please add assertion, otherwise this is not a test :) Small trap in task :)

File: pom.xml (hunk 1 above)
Comment (EugenBorisik) [reply pom.xml]: Optional: Also, I prefer selenium-java artefact, it contains all selenium staff for java

File: src/test/java/pages/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package pages;
+
+import org.openqa.selenium.By;
+
+public class LoginPage{
+
+    public static final String URL = "https://192.168.100.26/ ";
+    public By email = new By.ByCssSelector("#Username");
+    public By password = new By.ByCssSelector("#Password");
+    public By submit = new By.ByCssSelector("#SubmitButton");
```
Comment (EugenBorisik) [inline src/test/java/pages/LoginPage.java]: Optional: I prefer to use static methods from By class, f.e. By.xpath(), By.cssSelector, By.id, etc.

**Review by EugenBorisik — CHANGES_REQUESTED** [review]
Please update code according to comments + add chapter 2 from task

File: pom.xml (hunk 1 above)
Comment (DmitryMandrik) [reply pom.xml]: Thanks!

File: src/test/java/pages/LoginPage.java (hunk 3 above)
Comment (DmitryMandrik) [reply src/test/java/pages/LoginPage.java]: done

**Review by EugenBorisik — APPROVED** [review]

**Pull request merged**